TeamSnapSDK/SDK/TSDKCompletionBlockTypes.h

```diff
@@ -16,7 +16,8 @@ typedef void (^TSDKRootLinkCompletionBlock)(TSDKRootLinks *_Nullable rootLinks,
 typedef void (^TSDKCompletionBlock)(BOOL success, BOOL complete, TSDKCollectionJSON *_Nullable objects, NSError *_Nullable error);
 typedef void (^TSDKSaveCompletionBlock)(BOOL success, TSDKCollectionObject *_Nullable object, NSError *_Nullable error);
 typedef void (^TSDKInviteStatusCompletionBlock)(BOOL success, BOOL complete, TSDKinvitationFinder *_Nullable objects, NSError *_Nullable error);
-typedef void (^TSDKLoginCompletionBlock)(BOOL success, NSString *_Nullable OAuthToken, NSError *_Nullable error);
+typedef void (^TSDKTokenCompletionBlock)(BOOL success, NSString *_Nullable OAuthToken, NSError *_Nullable error);
+typedef void (^TSDKLoginCompletionBlock)(BOOL success, NSString *_Nullable OAuthToken, TSDKUser *_Nullable user, NSError *_Nullable error);
```

Comment:
@SkylerSeamans 
>  [[TSDKTeamSnap sharedInstance] loginWithOAuthToken:OAuthToken completion:^(BOOL success, NSError *error, TSDKUser *user)

I think user / Error are swapped between this, and root links #366